sonic_platform_base/sonic_ssd/ssd_generic.py

```diff
@@ -154,6 +159,19 @@ def parse_virtium_info(self):
                 except ValueError:
                     pass
 
+    def parse_swissbit_info(self):
+        if self.ssd_info:
+            health_raw = self.parse_id_number(SWISSBIT_HEALTH_ID)
+            if health_raw == NOT_AVAILABLE:
+                self.health = NOT_AVAILABLE
+            else:
+                self.health = health_raw.split()[-1]
+            temp_raw = self.parse_id_number(INNODISK_TEMPERATURE_ID)
```

Comment:
@Junchao-Mellanox this parsing seems to be for swissbit, why using innodisk id?

Reply:
Fixed